pkg/filesystem/fs/fs.go

```diff
@@ -336,6 +336,16 @@ func (fs *Filesystem) RemoveCache(blobDigest string) error {
 		return errors.Wrapf(err, "invalid blob digest from label %s", label.CRILayerDigest)
 	}
 	blobID := digest.Hex()
+	if fs.fsDriver == config.FsDriverFscache {
+		c, err := fs.getSharedDaemon().GetClient()
+		if err != nil {
+			log.L.Warnf("GetClient err %s", err)
```

Comment:
Without a successful client `c`, we can't call `UnbindBlob`, should return error here to containerd

Reply:
Actually, we call RemoveCache in a goroutine , the error will not pass to containerd.

Reply:
But we still have to return once GetClient() fails, otherwise, the whole process panics when performing L345

Reply:
Please don't ignore errors and assume that a function must run in a separate goroutine.

Reply:
sorry for confusion, actually my last reply is for the below comment~ . Yes, we can not ignore the err for getClinet. I mean we delete cache in a goroutine , then if get error and make cache residue, the error will not pass to containerd. so maybe we can just ignore the error if can not delete cache. I will update the code firstly , then we can move on.